hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -432,6 +432,14 @@
       base_addr: "0x40420000",
       attr: "templated",
     },
+    { name: "sysrst_ctrl_aon",
+      type: "sysrst_ctrl",
+      clock_srcs: {clk_i: "io_div4", clk_aon_i: "aon"},
+      clock_group: "powerup",
```

Comment:
the clock group probably doesn't need to be "powerup", i would probably move it to the the "secure" category. 
@esshiu 
@mdhayter 

I'm assuming the sysrst_ctrl clock is not something you want software to be able to accidentally disable is that right? This thing being able to run all the time is probably fairly important.

Reply:
Oh yeah that's right. Let's move this to `secure` then.